website/docs/ocl.md

```diff
@@ -152,16 +170,23 @@ This is the list of security (groups) defined in the service network. Each secur
     }
   },
   "compute": {
-    "vm": [{
-      "name": "my-vm",
-      "type": "t2.large",
-      "platform": "linux-x64",
-      "vpc": "$.network.vpc[0]",
-      "subnet": "$.network.subnet[0]",
-      "security": ["$.network.security[0]"],
-      "storage": ["$.storage[0]}"],
-      "publicly": true
-    }]
+    "vm": [
+      {
+        "name": "my-vm",
+        "type": "t2.large",
+        "platform": "linux-x64",
```

Comment:
seems platform is unneeded here.

Reply:
Agree, as the architecture information for `vm` will be carryied by `artifact_image`, `platform` is not needed any more.